frontend/src/bundles/common/components/header/components/navigation/navigation.tsx

```diff
@@ -25,10 +25,10 @@ const Navigation = ({ avatarUrl }: Properties): JSX.Element => {
                     </Link>
                 </li>
                 <li>
-                    <img
-                        src={avatarUrl ?? defaultAvatar}
-                        alt="avatar"
-                        className={styles['avatar']}
+                    <Avatar
+                        size="sm"
+                        email={email}
+                        avatarUrl={avatarUrl || ''}
```

Comment:
```suggestion
  ...(avatarUrl && { avatarUrl })
```
if our linter allows this)
but maybe will be better just change typings to string | null

Reply:
It doesn't allow to do so( 
So, I changed the type of avatarUrl to string | null